### ISSUE TITLE
File not found error when using the action

### ISSUE THREAD

**Issue by bdubaut** [agyemanjp/ci-checks-action#7]
Hi,

I'm trying to use this action to annotate a workflow run with linter errors on my project. Everything seems to be working fine, except that when I get to the step that runs the annotation action, I get this error:
```
##[error]File not found: '/home/runner/work/_actions/prmph/ci-checks-action/1.0.1/dist/index.js'
```

The project is an Elixir project, here is the job yaml:
```yaml
lint:
    name: Credo linter
    runs-on: ubuntu-latest
    env:
      MIX_ENV: test
    needs: build
    steps:
      - uses: actions/checkout@v2
      - name: Set up Elixir
        uses: actions/setup-elixir@v1
        with:
          elixir-version: '1.10.4' # Define the elixir version [required]
          otp-version: '22.3' # Define the OTP version [required]
      - name: Restore dependencies cache
        uses: actions/cache@v2
        with:
          path: deps
          key: ${{ runner.os }}-mix-${{ hashFiles('**/mix.lock') }}
          restore-keys: ${{ runner.os }}-mix-
      - uses: actions/download-artifact@v2
        with:
          name: _build
          path: ./_build
      - run: mix credo --strict --format=json > credo_results.json # output is put in the JSON file here
      - name: Annotate PR
        uses: prmph/ci-checks-action@1.0.1
        with:
          repo-token: "${{ secrets.GITHUB_TOKEN }}"
          checks: "lint:credo_results.json" # calling the checks on the file
```

Am I doing something wrong?

**Comment by bdubaut** [comment]
In case the problem comes from the results file format, here is an example of a file with one linter offense found:
```json
{
  "issues": [
    {
      "category": "readability",
      "check": "Credo.Check.Readability.ModuleDoc",
      "column": 11,
      "column_end": 20,
      "filename": "apps/test_credo/lib/test_credo.ex",
      "line_no": 1,
      "message": "Modules should have a @moduledoc tag.",
      "priority": 1,
      "trigger": "TestCredo"
    }
  ]
}
```

If there is no offenses, the file looks like this:
```json
{
  "issues": []
}
```

**Comment by agyemanjp** [comment]
Thanks for the report. A new version that should fix all these issue was published a few days ago. Please use that version like so:
```
- name: Annotate Checks
        uses: **agyemanjp/ci-checks-action@1.0.2**
        with:
          ghToken: ${{ secrets.GITHUB_TOKEN }}
          checks: "lint:.lint-report.json|test:.test-report.json"

```

**Comment by agyemanjp** [comment]
Also, I have published an eslint formatter (https://www.npmjs.com/package/eslint-formatter-json-standard) that can be used to output in correct format, in case your lint tool does not give the proper format.

**Comment by bdubaut** [comment]
@agyemanjp thanks a lot for looking into it! It seems that the JSON format that [Credo](https://github.com/rrrene/credo) returns is not in a format that the action expects:
```
Run agyemanjp/ci-checks-action@1.0.2
  with:
    ghToken: ***
    checks: lint:credo_results.json
  env:
    MIX_ENV: test
[before-after-hook]: "Hook()" repurposing warning, use "Hook.Collection()". Read more: https://git.io/upgrade-before-after-hook-to-1.4
##[error]Error processing requested check "lint"
	Error: Error parsing check script output
    at parse (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6771:15)
    at /home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6852:100
    at Generator.next (<anonymous>)
    at /home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6732:71
    at new Promise (<anonymous>)
    at module.exports.429.__awaiter (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6728:12)
    at run (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6820:12)
    at Object.429 (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:6899:1)
    at __webpack_require__ (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:24:31)
    at startup (/home/runner/work/_actions/agyemanjp/ci-checks-action/1.0.2/dist/index.js:43:19)
```

**Comment by agyemanjp** [comment]
Actually @bdubaut , if you are thinking of forking the action, then I think it is better is you write a formatter that formats the credo output properly. Doing so allows one action to be extended by various formatters and reporters, which is a better situation than having an action for each tool. You can take a look at my eslint formatter for ideas

**Comment by bdubaut** [comment]
@agyemanjp that's a good point. Also, I could also propose a change to the Credo project to how the JSON is structured.

What standard are you using for this JSON?

**Comment by agyemanjp** [comment]
The link to the JSON schema is:
https://gist.githubusercontent.com/agyemanjp/0f43de0639a7ec872e9ebcbe6166d5d9/raw/ccb90a9298561f2ba7c07ba6843b2b25244f9cf7/code-check-general.schema.json

This link is also on my eslint formatter page: https://www.npmjs.com/package/eslint-formatter-json-standard, and in the project files as well. If you clone that project, you can see how I validate and use the schema.

**Comment by bdubaut** [comment]
@agyemanjp Cool! I'll see what I can do 😉

**Comment by agyemanjp** [comment]
Should be fixed as of v2.0.4
